### PR TITLE
Feat/coordinates

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
image

image

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 1]
```diff
@@ -1188,6 +1199,19 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
+  private adjustCoordinates(): void {
+    const timelineActionsBandWidth = 404;
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1203]: It depends on the language due to the translation of 'From' and 'To', as well as whether the timeline has a selection

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 2]
```diff
@@ -1188,6 +1199,19 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
...
+    const scaleMaxWidth = 100;
+    const mapCanvas = document.getElementsByClassName('mapboxgl-canvas');
+    if (mapCanvas && mapCanvas.length > 0) {
+      const bbox = mapCanvas[0].getBoundingClientRect();
+      if (bbox) {
+        const width = bbox.width;
+        this.coordinatesHaveSpace = (width - timelineActionsBandWidth - scaleMaxWidth - 20) > 230;
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1210]: Why 20 ? Is it the dedired spacing between the coordinates and the timeline actions ?

**Review by QuCMGisaia — APPROVED** [review]

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1203]: Fixed, I took the max value

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1210]: I fixed it. I added the toggle button width and 3 sm-margins

**Pull request merged**